plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -199,11 +200,11 @@ export class NunjucksWorkflowRunner implements WorkflowRunner {
       additionalTemplateFilters: this.options.additionalTemplateFilters,
     });
 
+    let taskTrack;
```

Comment:
this declaration doesn't have to be outside of the try/catch does it?

Reply:
That's in an inner catch block inside the try though, or maybe i'm misreading it